### DISCUSSION TITLE
Geographic location of data hosting

### DISCUSSION THREAD

**Discussion by joebullardPA** [opengisch/QField#4146] [Q&A]
Hi,

We are looking into options for field data capture - one of the sticking points is the geographic location where data is hosted - I understand that for qfield cloud this is Switzerland. Is it possible to choose a different location due? For example the UK or an EU member state. I see from the FAQ page this is not an option when using the standard subscription - but guessing it would be feasible with private cloud?

**Comment by ZsanettMed** [comment]
Hi, 

Yes, all data is stored on servers hosted in Switzerland. And yes, it would be possible with self-hosted QfieldCloud.
